package-examples/cert-manager-basic/README.md

```diff
@@ -0,0 +1,66 @@
+# cert-manager-basic
+
+## Description
+
+CertManager kpt package.
+
+## Usage
+
+### Fetch the package
+
+```sh
+kpt pkg get git@github.com/googlecontainertools/kpt.git/package-examples/cert-manager-basic[@VERSION] cert-manager-basic
+```
+
+### View package content
+
+```sh
+kpt pkg tree cert-manager-basic
+```
+
+### Apply the package
+
+```sh
+kpt live init cert-manager-basic
+kpt live apply cert-manager-basic --reconcile-timeout=2m --output=table
+```
+
+### Steps used to create the package
```

Comment:
FYI @justinsb @bgrant0607 @yuwenma This section lists the steps I followed to create the initial package. I am using `app` label to split out the manifests into respective directories.